app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -389,20 +392,31 @@ object TestHelper {
 
     /**
      * Changes the default language of the entire device, not just the app.
+     * Only works on the Debug version, not allowed to change system settings in other production builds.
```

Comment:
I'd say, "Runs on Debug variant as we don't want to adjust Release permission manifests"

Reply:
done, thanks